src/main/query_profiler.cpp

```diff
@@ -136,8 +136,6 @@ void QueryProfiler::Finalize(ProfilingNode &node) {
 		Finalize(*child);
 		if (op_node.type == PhysicalOperatorType::UNION &&
 		    op_node.GetProfilingInfo().Enabled(MetricsType::OPERATOR_CARDINALITY)) {
-			op_node.GetProfilingInfo().metrics.operator_cardinality +=
-			    child->GetProfilingInfo().metrics.operator_cardinality;
```

Comment:
Should we remove this, or is this a special case for the `UNION` operator?

Reply:
Oops! I took this out while refactoring but forgot to put it back. Thanks for catching it.